File: API/Schemas/detail.py
from pydantic import BaseModel 
from decimal import Decimal
class Detail(BaseModel):
    idDetalleVenta:int | None = None
    idProducto:int
    marcaProducto:str
    descripcionProducto:str
    categoriaProducto:str
    cantidad:int
    precio:Decimal
    total:Decimal
    idVenta:int|None = None
    class Config:
        orm_mode = True
        allow_population_by_field_name = True
        arbitrary_types_allowed = True

File: Frontend/src/app/models/detail.ts
export interface Detail {
    idDetalleVenta?:number,
    idVenta?: number,
    idProducto?: number,
    marcaProducto: string,
    descripcionProducto:string,
    categoriaProducto:string,
    cantidad:number,
    precio:number,
    total?:number
}
